source/_footer.erb

```diff
@@ -1,5 +1,4 @@
-<hr class="hidden" />
 <div id="footer">
-  <p><small><%= partial 'license' %></small></p>
-  <p><small>Powered by <a href="http://middlemanapp.com/">Middleman</a>.</small></p>
+  <%= partial 'license' %>
```

Comment:
![image](https://user-images.githubusercontent.com/325384/93709926-6470a400-fb3a-11ea-812a-c42b26d5855b.png)

The partial render markdown, which lead to a `<p>` within a `<p>`. 

I decided the right solution there was to just adjust the font-size in the footer to be the same as small.